test/required/requiredexamplepb_test.go

```diff
@@ -0,0 +1,69 @@
+package required
+
+import (
+	"github.com/gogo/protobuf/proto"
+	"testing"
+)
+
+func TestMarshalToErrorsWhenRequiredFieldIsNotPresent(t *testing.T) {
```

Comment:
Sorry, I'm used to a coding style where we break up the code more. I apologize for not doing a better job conforming to the style in the library.